### PR TITLE
Aa sticky header

### PR DESCRIPTION
**Pull request by dlangston**
@DanielJMaher @FBRTMaka @birdage 
before...
screen shot 2015-10-01 at 3 18 45 pm

after...
screen shot 2015-10-01 at 3 18 14 pm

### CODE REVIEW

File: ooiui/templates/aa/AlertPage.html [hunk 1]
```diff
@@ -49,30 +49,40 @@
...
+                <div id="alerts-panel" class="panel">
+                  <div id="alerts-search" class="panel-header">
+                    <div class="scrollable-block">
+                      <div class="scrollable-area" style="width: 100%; max-height: 305px; border: 2px solid #fff; overflow: auto;">
+                        <div class="panel-body">
+                          <div id='alertslist' class="col-sm-12">
+                          </div>
+                          <div>
+
+                            {#<div class="col-sm-12" id="action_buttons" style='font-weight:bold;padding-top:15px;'>
```
Comment (birdage) [inline ooiui/templates/aa/AlertPage.html]: @dlangston whats this? `{#`...?

**Comment by dlangston** [comment]
just commented out code. I'll go back and remove it.

**Comment by dlangston** [comment]
@birdage all set

File: ooiui/static/css/common/status.css [hunk 2]
```diff
@@ -548,6 +548,16 @@ BG
     content: "\e080";    /* adjust as needed, taken from bootstrap.css */
 }
 
+/************************/
+/****Alerts & Alarms****/
+/**********************/
+div#allerts-scrollable {
```
Comment (birdage) [inline ooiui/static/css/common/status.css]: @dlangston is it meant to be `allerts` or `alerts`?

**Comment by birdage** [comment]
@FBRTMaka hold on this till i check it please

**Comment by FBRTMaka** [comment]
@birdage you got it, all yours!

**Comment by dlangston** [comment]
@birdage @FBRTMaka fixed spelling and added css file

**Comment by birdage** [comment]
@oceanzus am i ok to merge it? are the changes going to affect you?

**Pull request merged**